firmware/tunerstudio/rusefi.input

```diff
@@ -924,16 +924,20 @@ menuDialog = main
 		
 	menu = "Injection"
 		subMenu = injectionSettings,		"Injection Settings"
+		subMenu = injectionDeadTime,		"Injector Dead Time"
 		subMenu = std_separator
 		subMenu = fuelTableDialog,			"Fuel Table", 0, {fuelAlgorithm != LM_SPEED_DENSITY}
 		subMenu = veTableDialog,			"VE Table", 0, {fuelAlgorithm == LM_SPEED_DENSITY}
 		subMenu = afrTableTbl,				"Target AFR Table", 0, {fuelAlgorithm == LM_SPEED_DENSITY}
-		subMenu = injPhaseTableTbl,			"Injection Phase"
+		subMenu = injPhaseTableTbl,         "Injection Phase Table"
```

Comment:
Name unification - all menu items with tables should have 'Table' in their names.